src/Omnipay/Realex/Message/VoidRequest.php

```diff
@@ -97,6 +95,10 @@ protected function createResponse($data)
 
     public function getEndpoint()
     {
-        return $this->endpoint;
+        return $this->getParameter('SecureDataVaultEndpoint');
+    }
+    public function setAuthEndpoint($value)
+    {
+        return $this->setParameter('SecureDataVaultEndpoint', $value);
```

Comment:
Should getters/setters not be using lower-camel-case parameter names to match the defaults defined in RemoteGateway.php? I'm not sure if the parameter bag is case-insenstive, but it would seem to make sense to be consistent with the other getters/setters...